jsapp/js/account/routes.tsx

```diff
@@ -84,6 +87,26 @@ export default function routes() {
           </RequireAuth>
         }
       />
+      {enableMMORoutes && (
```

Comment:
🤔 Good one...on mine I left the route active and I'm only hiding the tab item, so I can still access it even without the flag... but thinking about it now, it would be better to do what you here and 'hide' the route!